### ISSUE TITLE
Project links could be more visually accessible.

### ISSUE THREAD

**Issue by dengeist** [dengeist/dengeist.github.io#1]
Instead of list items with anchor tags and plain text, these should be `button` elements or at least button-like in their display. This might also help with ARIA navigability, though I have not directly tested that.
